lib/msal-node/src/cache/CacheContext.ts

```diff
@@ -16,18 +16,17 @@ import { Storage } from '../cache/Storage';
  * class managing sync between the persistent cache blob in the disk and the in memory cache of the node
  */
 export class CacheContext {
-
     private cachePath: string;
     private defaultSerializedCache: JsonCache = {
         Account: {},
         IdToken: {},
         AccessToken: {},
         RefreshToken: {},
-        AppMetadata: {}
+        AppMetadata: {},
     };
 
     constructor() {
-        this.cachePath = "";
+        this.cachePath = '';
```

Comment:
Do we need cachePath anymore?

Reply:
We don't if we have keyTar working as path is calculated. Also I think Daryl's PR makes this unnecessary. I can pull dev and change this.